core/lib/libtomcrypt/cmac.c

```diff
@@ -82,10 +83,16 @@ static const struct crypto_mac_ops ltc_omac_ops = {
 	.copy_state = ltc_omac_copy_state,
 };
 
-TEE_Result crypto_aes_cmac_alloc_ctx(struct crypto_mac_ctx **ctx_ret)
+static TEE_Result crypto_common_cmac_alloc_ctx(struct crypto_mac_ctx **ctx_ret,
+		const char *const cipher)
 {
+	int cipher_idx;
 	struct ltc_omac_ctx *ctx = NULL;
-	int cipher_idx = find_cipher("aes");
+
+	if (!cipher || !ctx_ret)
```

Comment:
Don't bother checking `cipher`, this is a static function and we know all callers pass non-NULL. So:
```C
int cipher_idx = find_cipher(cipher);
```

Reply:
Done as requested.